mllib/src/main/scala/org/apache/spark/ml/r/MultilayerPerceptronClassifierWrapper.scala

```diff
@@ -73,6 +75,8 @@ private[r] object MultilayerPerceptronClassifierWrapper
       .setStepSize(stepSize)
       .setPredictionCol(PREDICTED_LABEL_COL)
     if (seed != null && seed.length > 0) mlp.setSeed(seed.toInt)
+    if (initialWeights != null) mlp.setInitialWeights(Vectors.dense(initialWeights))
```

Comment:
that seems somewhat involved, so it is likely better to not duplicate the logic on the R side.
we should check here if initialWeights != null its length > 0